gradle.properties

```
@@ -3,19 +3,19 @@ java_version=17
 
 minecraft_version=1.20.1
 mappings=1
-enabled_platforms=fabric,forge
+enabled_platforms=fabric
 
 # Change these to your plugin id, version number and org or maven group, it must be unique.
-archives_base_name=examplefiguraplugin
+archives_base_name=goofyfiguraplugin
 mod_version=1.0
 maven_group=org.figuramc
```

needs to be changed to new class
```suggestion
maven_group=com.thekillerbunny.goofyplugin
```